### ISSUE TITLE
no XML docs on anything

### ISSUE THREAD

**Issue by brandonh-msft** [Microsoft/botbuilder-dotnet#454]
There are no XML documentation tags on any of the APIs in the m3.1 nupkgs.
![microsoftteams-image](https://user-images.githubusercontent.com/20270743/38946568-b8c4a2ba-42ee-11e8-892f-cbd9d6be7d20.png)
![microsoftteams-image 1](https://user-images.githubusercontent.com/20270743/38946590-cd657ec4-42ee-11e8-9f60-b7663f4024df.png)

**Comment by drub0y** [comment]
Yeah, the problem is the `<DocumentationFile>` property has a `Condition` to only build for the `Documentation` specific build configuration only and not `Release`. There's a couple ways to skin this cat. Options:

 1. Put the `<DocumentationFile>` property into the `Release` specific property group - probably simplest for now, but dupes it
 2. Change the condition on the current property group that was created just for `<DocumentationFile>` so that it evals to true for `Release` config too - gettin' messy
 3. Introduce a custom property called maybe `ShouldBuildDocs` with a value of `true`, put that into the build configuration property groups that want docs to be built, then just put <DocumentationFile> into the base property group and put a condition on it of `Condition="'$(ShouldBuildDocs)'!=''"`.

Finally, I would suggest we also parameterize the `<DocumentationFile>` property instead of being so hardcoded, so the final property might look like:

```XML
<DocumentationFile Condition="'$(ShouldBuildDocs)'!=''">$(OutputPath)$(AssemblyName).xml</DocumentationFile>
```
